### PR TITLE
animations, gallery, and colors

### PR DESCRIPTION
**Pull request by RedCat888**
I edited color of links and made </hr/> slightly thicker. New animations in navbar, but took a lot of time to make it not mess with the mobile version/setup of navbar. Also added gallery section on our main page ("home") which shows some design concepts that show off the club and add some real content.

### CODE REVIEW

**Comment by RedCat888** [comment]
https://github.com/user-attachments/assets/92b63c5b-cf58-46c7-a401-83dac23e4235

**Comment by Blue-Robin-Taken** [comment]
Vercel deployment fails...

**Comment by Blue-Robin-Taken** [comment]
`10:34  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities`

File: src/app/page.tsx
Comment (Blue-Robin-Taken) [inline src/app/page.tsx:1]: The apostrophe on the `<h1>` here causes the vercel deployment to fail
instead escape this character `&apos;`

**Review by Blue-Robin-Taken — CHANGES_REQUESTED** [review]
Deployment must work for me to approve this pull request

**Review by Blue-Robin-Taken — APPROVED** [review]
Deployment works now

**Pull request merged**